frontend/src/components/TreeViewer/TreeViewer.jsx

```diff
@@ -235,18 +240,32 @@ function toggleInfoBoxVisibility(clickedNodeName, curViewingNodeName) {
   return nodeViewingAfterToggle;
 }
 
-export const TreeViewer = ({ data, treeID }) => {
+/**
+ * Gets the arguments of the query string from a given URL
+ * @param {string} name The name of the query-string parameter to get
+ * @param {string} url The url from where to parse query string
+ */
+const getParameterByName = (name, url) => {
```

Comment:
From @mit1221 : "We should probably add the base address here ("http://localhost:3003") and all other uses of it to the config file" Source: https://github.com/VerWiki/VerWikiApp/pull/22#discussion_r611084902

Reply:
Done